HiveCorePlugins.sln

```diff
@@ -36,11 +38,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools", "tools", "{85BD8051
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Hive.PermissionQuery", "src\Hive.PermissionQuery\Hive.PermissionQuery.csproj", "{72879465-FE87-4F28-839D-A9A6A1922D50}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.Webhooks", "src\Hive.Webhooks\Hive.Webhooks.csproj", "{252F0961-543A-40D6-8C85-1823FD95A36C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Hive.Webhooks", "src\Hive.Webhooks\Hive.Webhooks.csproj", "{252F0961-543A-40D6-8C85-1823FD95A36C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.AdditionalUserDataExposer", "src\Hive.AdditionalUserDataExposer\Hive.AdditionalUserDataExposer.csproj", "{4CB8645F-0901-4D86-BAAD-D0E5E9ED6E64}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Hive.AdditionalUserDataExposer", "src\Hive.AdditionalUserDataExposer\Hive.AdditionalUserDataExposer.csproj", "{4CB8645F-0901-4D86-BAAD-D0E5E9ED6E64}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.Tags.Categories", "src\Hive.Tags.Categories\Hive.Tags.Categories.csproj", "{11DF3A46-C69A-4090-A0A6-CE2FAFF4FF08}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Hive.Tags.Categories", "src\Hive.Tags.Categories\Hive.Tags.Categories.csproj", "{11DF3A46-C69A-4090-A0A6-CE2FAFF4FF08}"

```

Comment:
Why did these hashes change? New project?

Reply:
I didnt purposefully change these GUIDs. Probably changed in some internal or automated process. Could be reverted.